core/src/components/item/item.ionic.scss

```
@@ -0,0 +1,113 @@
+    * @prop --border-radius: Radius of the item border
+    * @prop --border-style: Style of the item border
+    * @prop --border-width: Width of the item border
+    *
+    * @prop --color: Color of the item
+    * @prop --color-activated: Color of the item when pressed
+    * @prop --color-focused: Color of the item when focused with the tab key
+    * @prop --color-hover: Color of the item on hover
+    *
+    * @prop --detail-icon-color: Color of the item detail icon
+    * @prop --detail-icon-opacity: Opacity of the item detail icon
+    * @prop --detail-icon-font-size: Font size of the item detail icon
+    *
+    * @prop --inner-border-width: Width of the item inner border
+    * @prop --inner-box-shadow: Box shadow of the item inner
+    * @prop --inner-padding-top: Top padding of the item inner
+    * @prop --inner-padding-end: Right padding if direction is left-to-right, and left padding if direction is right-to-left of the item inner
+    * @prop --inner-padding-bottom: Bottom padding of the item inner
+    * @prop --inner-padding-start: Left padding if direction is left-to-right, and right padding if direction is right-to-left of the item inner
+    *
+    * @prop --min-height: Minimum height of the item
+    *
+    * @prop --padding-bottom: Bottom padding of the item
+    * @prop --padding-end: Right padding if direction is left-to-right, and left padding if direction is right-to-left of the item
+    * @prop --padding-start: Left padding if direction is left-to-right, and right padding if direction is right-to-left of the item
+    * @prop --padding-top: Top padding of the item
+    *
+    * @prop --transition: Transition of the item
+    *
+    * @prop --ripple-color: Color of the item ripple effect
+    *
+    */
+  --border-color: #{globals.$ionic-color-neutral-200};
+  --border-style: #{globals.$ionic-border-style-solid};
+  --border-width: #{globals.$ionic-border-size-025};
+  --default-background: #{globals.$ionic-color-base-white};
+  --item-min-height: #{globals.$ionic-scale-1800};
+  --item-width: #{globals.$ionic-line-height-full};
+  --space-200: #{globals.$ionic-space-200};
+  --space-400: #{globals.$ionic-space-400};
+
+  @include globals.font-smoothing;
+
+  display: flex;
+
+  position: relative;
+
+  align-items: center;
+  justify-content: space-between;
+
+  outline: none;
+
+  font-family: globals.$ionic-font-family;
+
+  text-align: initial;
+  text-decoration: none;
+
+  overflow: hidden;
+  box-sizing: border-box;
+}
+
+// Item: Structure
+// --------------------------------------------------
+.item-native {
+  @include globals.padding(var(--space-200), var(--space-400));
+  display: flex;
+
+  align-items: center;
+
+  width: var(--item-width);
+  min-height: var(--item-min-height);
+
+  border-bottom-width: var(--border-width);
+  border-bottom-style: var(--border-style);
+  border-bottom-color: var(--border-color);
+
+  background: var(--default-background);
+
+  overflow: inherit;
```

```suggestion
  overflow: inherit;
  
  box-sizing: border-box;
```

Item is expanding height due to the padding without this